scss/aSideShowTimes.scss

```diff
@@ -0,0 +1,113 @@
+.showtime {
```

Comment:
You're making my eyes hurt with asideShowTimes.js and aSideShowTimes.scss 🤣

Reply:
In a good or badway o.O

Reply:
Since I'm not a masochist, my eyes hurting will always be bad 😉

Reply:
Well I didn't want to assume anything, too many comments or to messy code ?

Reply:
Noo! Just the capital S in aSideShowTimes and lowercase in asideShowTimes 😂

Reply:
Haha, sry xD
Fixed it now!